DependencyInjection/MonologExtension.php

```diff
@@ -48,6 +51,10 @@ private function levelToMonologConst($level)
      */
     public function load(array $configs, ContainerBuilder $container)
     {
+        if (class_exists(FullStack::class) && Kernel::MAJOR_VERSION < 5 && Logger::API >= 2) {
```

Comment:
Why this check on `FullStack` ?

Reply:
I want to detect that case that Monolog 2 is installed together with `symfony/symfony` 2.x/3.x/4.x. I don't see a good way to make Composer prevent that combination, so I'm catching that case here. If you know a better way, I'm open for suggestions.

Reply:
I think the point is the Kernel::MAJOR_VERSION is enough? Or why do you need three datapoints to check two versions?

Reply:
`Kernel::MAJOR_VERSION` is the version of the HttpKernel component, not the version of MonologBridge. But it's the bridge that is actually incompatible before version 5. I have no way to check for the bridge version programmatically. The assumption that the versions of MonologBridge and HttpKernel match is only valid if we have the monorepo.

If I removed the monorepo check, we would also throw if the app uses the standalone packages HttpKernel 4.4, MonologBridge 5.0 and Monolog 2, but that combination should work fine.

Reply:
Ok gotcha, then sounds good to me to merge as is.. @lyrixx ?

Reply:
Why do we allow symfony/symfony 4.4 and monolog/monolog 2 in composer, but throw an exception manually? 
I mean, We could forbid installation thanks to composer, isn't ?

Reply:
That was turned down in https://github.com/symfony/symfony/pull/32469 - IMO this is fine, people using the bundle with the wrong version of symfony+monolog will get an exception, then they can fix.. Not ideal but not so bad either..

Other alternative is to require Monolog 2 and Symfony 5 or at least the monolog bridge in v5 then we fork the bundle completely in two branches for monolog 1 and 2 support.

Reply:
:+1: